src/Modules/Release.php

```diff
@@ -78,7 +78,23 @@ public function create(&$force, &$deploy_public)
 		// Clone WP and remove things we don't want
 		$wp = new \Dxw\Whippet\Git\Git($this->release_dir);
 		$wp->clone_repo($this->application_config->wordpress->repository);
-		$wp->checkout($this->application_config->wordpress->revision);
+
+		if (getenv('WP_CONFIG_WP_ENVIRONMENT_TYPE') === false) {
```

Comment:
Do we need to tighten this up? e.g. if `WP_CONFIG_WP_ENVIRONMENT_TYPE` was set to `production` on a prod site, this would result in us deploying the latest version there (assuming the "latest" tag existed).

Reply:
sorry, forgot to rebase. I've pushed a fixup commit